jaas-ljm/example.js

```diff
@@ -208,46 +317,53 @@ function addRegionsOptions() {
         optionElem.text = upper;
         regionSelectElem.appendChild(optionElem);
     });
-}
+};
 
-$(window).bind('beforeunload', disconnect);
-$(window).bind('unload', disconnect);
+const handleReleaseUpdate = async event => {
+    if ((!releaseVersion && !event.target.value) || releaseVersion === event.target.value) {
+        return;
+    }
 
-$(document).ready(function() {
-    addRegionsOptions();
-    $("#goButton").click(async function() {
-        const tenant = $("#tenantInput").val();
-        token = $("#tokenInput").val();
-        roomName = $("#roomInput").val();
+    releaseVersion = event.target.value;
+    const releaseVersionParam = event.target.value ? `?release=release-${event.target.value}` : '';
+    const currentVersionScript = document.getElementById(LJM_SCRIPT);
+    let nextVersionScript = document.createElement('script');
 
-        options = buildOptions(tenant, roomName);
+    nextVersionScript.id = LJM_SCRIPT;
+    nextVersionScript.src = `${BASE_SOURCE}${releaseVersionParam}`
 
-        // Initialize lib-jitsi-meet
-        JitsiMeetJS.init(options);
+    currentVersionScript.remove();
+    document.body.appendChild(nextVersionScript);
```

Comment:
Is this enough to unload JitsiMeetJS completely? Do you see the version changing?

Reply:
Yes, it prints a different commit hash after reload.